tests/test_browser.py

```diff
@@ -4013,8 +4020,11 @@ def test_emscripten_set_canvas_element_size(self):
 
   # Tests that emscripten_run_script() variants of functions work in pthreads.
   def test_pthread_run_script(self):
-    for args in [[], ['-s', 'USE_PTHREADS=1', '-s', 'PROXY_TO_PTHREAD=1']]:
-      self.btest(path_from_root('tests', 'pthread', 'test_pthread_run_script.cpp'), expected='1', args=['-O3', '--separate-asm'] + args, timeout=30)
+    self.btest(path_from_root('tests', 'pthread', 'test_pthread_run_script.cpp'), expected='1', args=['-O3', '--separate-asm'], timeout=30)
+
+  @requires_threads
+  def test_pthread_run_script_pthreads(self):
```

Comment:
Hmm I like that these tests are getting split, but now I find this test name a little confusing.    Should they be called `test_run_script` and `test_run_script_pthread`?  Although then test_pthread_run_script.cpp is strangely named?   Perhaps this test doesn't need to be split, only marked, and perhaps it doesn't need the first `btest` at all?

Reply:
I think the original intent was to see that this pthreads-related stuff works in both pthreads and non-pthreads builds. That is, pthreads API stuff can be used in a single-threaded build too.

But yeah, the name seems slightly odd, I agree, given the double meaning of "pthreads" here... not sure I have a good idea to improve it though :)

Reply:
(concretely, the pthreads stuff tested here are global vars like `ENVIRONMENT_IS_PTHREAD`, which should exist and be valid in non-multithreaded builds too)